nvim/after/plugin/zenmode.lua

```diff
@@ -0,0 +1,14 @@
+require('zen-mode').setup({
+  window = {
+    width = 90,
+    options = {
+      number = true,
+      relativenumber = true,
+    },
+  },
+})
+
+vim.keymap.set('n', '<leader>zz', function()
+  require('zen-mode').toggle()
+  vim.wo.wrap = false
+end)
```

Comment:
For some reason, this map gets blown away... I'm guessing it is which-key and it having a `z` default, but I don't know.